PERSONAL_DATA_DISCLOSURE.md

```diff
@@ -0,0 +1,84 @@
+_Last Updated: Mon, 17 Oct 2022 UTC
+_Last Reviewed: Mon, 17 Oct 2022 UTC
+
+## Overview
+
+The purpose of this form is to disclose the types of personal data[^1] (PD) stored by each module.  This information enables those hosting FOLIO to better manage and comply with various privacy laws and restrictions, e.g. GDPR.
+
+It's important to note that PD is not limited to that which can be used to identify a person on it's own (e.g. Social security number), but also data used in conjunction with other data to identify a person (e.g. date of birth + city + gender, or dynamic IP address + date + time), and any information about such a person (e.g. has loaned 5 items, 2 of which are overdue).
+
+For the purposes of this form, "store" includes the following:
+
+* Persisting to storage - Either internal (e.g. Postgres) or external (e.g. S3, etc.) to FOLIO
+* Caching - In-memory, etc.
+* Logging
+* Sending to an external piece of infrastructure such as a queue (e.g. Kafka), database (e.g. Elasticsearch, Library Data Platform), distributed table, etc.
+
+## Personal Data processed by this Module
+
+- [ ] This module does not store any PD.
+- [ ] This module does not process any PD.
+- [ ] This module provides [custom fields](https://github.com/folio-org/folio-custom-fields).
+- [x] This module stores fields with free-form text (tags, notes, descriptions, etc.)
```

Comment:
What free form information does this module store?

Reply:
A **Charge** has a `description`, a `state`, and a `reason` field/property.

A **Money** (and also **TotalCharges**) has a `isoCurrencyCode` that feels like it should be restricted or limited but it appears to have no actual restrictions. I would consider this specific case a grey-area in the free-form text category.

A **Hold** has a `cancellationAdditionalInformation`, and a `patronComments`.

An **Item** has a `title` (a grey area), an `author`  (a grey area), and an `isbn` (a grey area).

Reply:
Are those records persistently stored by this module?

I reckon this might come down to the ambiguity around the term `store` in this document.

Reply:
> Which is essentially saying "transmits"

I agree this is vague and confusing.

The way I understand the motivation for this is that infrastructure is separate to the module and could / may persistently store that store. 

Tools like Kafka provide persistent storage, even if it is considered shorter lived than something like PostgreSQL. I agree this is highly dependent upon how those tools are used and that leads to a bunch of ambiguity.

Are you suggesting that we should include any transmission of data be considered as storage e.g. this module sends on this data to a client?

Reply:
> Are you suggesting that we should include any transmission of data be considered as storage e.g. this module sends on this data to a client?

That is how I have interpreted the PDD.
I would much prefer to have `store` mean the traditional message or have the message clarified such as appending
"... for the purposes of storing." (which can still be a little vague but it is an improvement).

That is, something more like this:
```
Sending to an external piece of infrastructure for the purpose of storing, such as a queue (e.g. Kafka), database (e.g. Elasticsearch, Library Data Platform), distributed table, etc.
```

If you believe that is how this is intended to be read, then I have no objections in regards to changing the checkboxes that I have checked.